javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,161 @@
+ 
+ // full name function
+ function getFullname(firstName, surname,useFormalName=false,gender){
+    if(useFormalName){
+        if(gender==="female"){
+          return "Queen " +firstName+" "+surname;
+        }
+        else{
+            return "Lord " +firstName+" "+surname;
+        }
+    }
+    else
+    return firstName+" "+surname;
+ }
+  const fullname1={
```

Comment:
write a function and utilize objects directly inside. they are instead picking each key pair during function execution.